all.json

```diff
@@ -25,6 +25,18 @@
 		"2ethereum.website",
 		"2polkadot.com",
 		"2rfleche.ma",
+		"cake-defi.net",
+		"coinhamsterdrop.com",
+		"collab-land-verify.com",
+		"collablandconnect.net",
+		"collablandsync.net",
+		"collablandupdate.com",
+		"dapps-solution.com",
+		"dapps-solution.com.rectificationwal.online",
```

Comment:
This seems listed? `rectificationwal.online` is blocked.

Reply:
ah, true will remove